scripts-components/toggler/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, EventListeners, DelegateEvent } from '@mangoweb/scripts-base'
+
+interface TogglerData {
+	targetSelector: string,
+	defaultClassName?: string
+}
+
+export default class Toggler extends Component<TogglerData> {
+	static componentName = 'Toggler'
+
+	getListeners = (): EventListeners => [['click', this.props.targetSelector, this.handleClick]]
+
+	handleClick(e: DelegateEvent<'click'>) {
+		const delegateTarget = e.delegateTarget
+		const targetSelector = delegateTarget.dataset.target
```

Comment:
`targetSelector` se nebere s `this.props.targetSelector` nebo pomocí `this.getPropOrElse('targetSelector', 'default')`